src/index.js

```diff
@@ -1 +1,4 @@
-require('./styles/fonts.scss')
+require('./scripts/accidenz_commons')
```

Comment:
Based on conversations with the team, `accidenz_commons` isn't used for text anywhere at the moment. So I think we can remove this from the main bundle

Reply:
Everything else looks good 👍

Reply:
Resolved in 2ea0024.